alvr/audio/src/windows.rs

```diff
@@ -69,3 +72,8 @@ pub fn set_mute_windows_device(device: &AudioDevice, mute: bool) -> Result<()> {
 
     Ok(())
 }
+
+pub fn is_same_device(device1: &AudioDevice, device2: &AudioDevice) -> bool {
+    device1.inner.name() == device2.inner.name()
+        && device1.inner.supports_output() == device2.inner.supports_output()
+}
```

Comment:
Is this function here by mistake? Also in an earlier PR you just simply used cpal eq operator

Reply:
> Probably doesn't work without as_inner()

Well yeah 😅